resources/views/users/show.blade.php

```diff
@@ -0,0 +1,211 @@
+@extends('layouts.master')
+@section('main-content')
+    @if(!isset($user))
+        <h1>{{ __('Nothing to show!') }}</h1>
+    @else
+        <div class="container bootstrap snippet">
+            <div class="panel-body inf-content">
+                <div class="row">
+                    <div class="col-md-4">
+                        @if(!isset($user->image))
+                        <img alt="" title="" class="img-circle img-thumbnail isTooltip"
+                             src="https://bootdey.com/img/Content/user-453533-fdadfd.png" data-original-title="Usuario">
+                        @else
+                            <img alt="" style="width:600px;" title="" class="img-circle img-thumbnail isTooltip"
+                                 src="/images/users/{{ $user->image }}" data-original-title="Usuario">
+                        @endif
+                    </div>
+                    <div class="col-md-6">
+                        <strong>{{ __('User Information') }}</strong><br>
+                        <div class="table-responsive">
+                            <table class="table table-condensed table-responsive table-user-information">
+                                <tbody>
+                                <tr>
+                                    <td>
+                                        <strong>
+                                            <span class="glyphicon glyphicon-asterisk text-primary"></span>
+                                            Identificacion
+                                        </strong>
+                                    </td>
+                                    <td class="text-primary">
+                                        {{ $user->id }}
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td>
+                                        <strong>
+                                            <span class="glyphicon glyphicon-user  text-primary"></span>
+                                            {{ __('Full Name') }}
+                                        </strong>
+                                    </td>
+                                    <td class="text-primary">
+                                        {{ $user->full_name }}
+                                    </td>
+                                </tr>
+
+                                <tr>
+                                    <td>
+                                        <strong>
+                                            <span class="glyphicon glyphicon-bookmark text-primary"></span>
+                                            {{ __('Gender') }}
+                                        </strong>
+                                    </td>
+                                    <td class="text-primary">
+                                        {{ $user->gender == 1 ? __('Male') : __('Female') }}
```

Comment:
gender is boolean type (has description in branch migration), gender = 0 -> female, gender = 1 -> male